firmware/1.5.4.1/app/include/user_config.h

```diff
@@ -97,7 +97,7 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
 #define LUA_PROCESS_LINE_SIG 2
 #define LUA_OPTIMIZE_DEBUG      3
 
-#define ENDUSER_SETUP_AP_SSID "AlarmPanel"
+#define ENDUSER_SETUP_AP_SSID "konnected-io"
```

Comment:
This variable I believe is only for the first time wifi setup. This is the WiFi AP name that the enduser-setup module broadcasts initially when wifi is not configured. I don't think there is an option to append the mac identifier here.